packages/react-liveness/src/components/FaceLivenessDetector/FaceLivenessDetectorCore.tsx

```diff
@@ -43,6 +43,21 @@ export default function FaceLivenessDetectorCore(
     },
   });
 
+  const [state, send] = useActor(service);
+  const isStartView = state.matches('start');
+
+  const beginLivenessCheck = React.useCallback(() => {
+    send({
+      type: 'BEGIN',
+    });
+  }, [send]);
+
+  React.useLayoutEffect(() => {
```

Comment:
Why is `useLayoutEffect` needed here? (as opposed to `useEffect`)

Reply:
Ah no particular reason, i think this was just copied from somewhere else in the code which required useLayoutEffect, good catch!